roles/ovirt-datacenter-cleanup/README.md

```diff
@@ -17,6 +26,9 @@ Role Variables
 |--------------------------|-----------------------|--------------------------------------|
 | data_center_name         | UNDEF                 | Name of the data center.             |
 | format_storages          | false                 | Whether role should format storages when removing them. |
+| hosted_engine            | false                 | Specify whether we datacenter to cleanup contains a VM which is serving an OVirt hosted engine installation.|
+| hosted_engine_vm_name    | HostedEngine          | Name of the hosted engine VM. Valid only when hosted_engine == true. |
+| hosted_engine_sd_name    | hosted_storage        | Name of the storage domain in which the Hosted engine VM resides on. Valid only when hosted_engine == true. |
```

Comment:
Should we automatically detect the storage of `hosted_engine` or are there any issues with that?

Reply:
I am not aware how to do it. I could not get such info with engine-config.
I can get however the hosted_engine_vm name with engine-config. But I 'll need some regexp there to extract it and i thought it would look ugly.
What do you think?